FlowCrypt/Core/Core.swift

```diff
@@ -27,71 +27,94 @@ protocol KeyParser {
     func parseKeys(armoredOrBinary: Data) async throws -> CoreRes.ParseKeys
 }
 
-actor Core: KeyDecrypter, KeyParser, CoreComposeMessageType {
+class Core: KeyDecrypter, KeyParser, CoreComposeMessageType {
     static let shared = Core()
 
     private typealias CallbackResult = (String, [UInt8])
 
-    private var jsEndpointListener: JSValue?
-    private var cb_catcher: JSValue?
-    private var vm = JSVirtualMachine()!
-    private var context: JSContext?
-
-    private var callbackResults: [String: CallbackResult] = [:]
-    private var ready = false
+    private lazy var webView: WKWebView = {
+        let userController = WKUserContentController()
+        userController.add(coreMessageHandler, name: "coreHost")
+        let configuration = WKWebViewConfiguration()
+        configuration.userContentController = userController
+        return WKWebView(frame: .zero, configuration: configuration)
+    }()
 
     private lazy var logger = Logger.nested(in: Self.self, with: "Js")
+    private let coreMessageHandler = CoreMessageHandler()
+
+    private struct RawRes {
+        let json: Data
+        let data: Data
+    }
+
+    private struct GmailBackupSearchResponse: Decodable {
+        let query: String
+    }
 
-    private init() {}
+    private init() {
+        setupWebView()
+    }
+
+    // MARK: - Setup
+    func setupWebView() {
+        guard let jsFileSrc = getCoreJsFile() else { return }
+
+        let appVersion = Bundle.main.infoDictionary?["CFBundleShortVersionString"] as? String ?? "1.0.0"
```

Comment:
the `??` should be `[unknown version]` or similar

Reply:
Done ✅